hack/verify-import-restrictions.sh

```diff
@@ -25,6 +25,16 @@ set -o pipefail
 
 sub_packages=(
```

Comment:
Wondering if we can say to run it just anywhere and respect the config files? Or determine the paths to test by `find -name ".import-restrictions"`

Reply:
Let's maybe try to optimize this in a follow up

Reply:
> Wondering if we can say to run it just anywhere and respect the config files?

I have not tested it by removing the sub package, I can see how it behaves.

> Or determine the paths to test by find -name ".import-restrictions"

Otherwise it is ok to do this as well.

Reply:
I'm fine with optimizing if we find a save way to do it. I would just like to have the restrictions in place :) (so I don't want to wait on finalizing the scripting :))

Reply:
Yeah a follow up PR sounds good. I don't have much idea about this myself 😅

Reply:
I would say it's fine as is. @chrischdi feel free to open a follow-up PR if you want to improve it :)